### ISSUE TITLE
WCAG A 4.1.2 Issue on Playlist View Page

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#6427]
## Description
On a playlist view page in avalon-dev (ex: https://avalon-dev.dlib.indiana.edu/playlists/96), a "Hidden element has focusable content" error shows in the Siteimprove browser plugin (Chrome).

## Done Looks Like
- [ ] Aria attribute issue causing the 4.1.2 violation is fixed
- [ ] Playlist view page meets WCAG 2.1 AA (should be the case after this is resolved)

**Comment by joncameron** [comment]
Resolved—the playlist view page now meets WCAG 2.1 AA!
